pkg/controller/constraint/constraint_controller.go

```diff
@@ -65,7 +65,7 @@ import (
 var (
 	log                 = logf.Log.V(logging.DebugLevel).WithName("controller").WithValues(logging.Process, "constraint_controller")
 	discoveryErr        *apiutil.ErrResourceDiscoveryFailed
-	DefaultGenerateVAPB = flag.Bool("default-create-vap-binding-for-constraints", false, "Create VAPBinding resource for constraint of the template containing VAP-style CEL source. Allowed values are false: do not create Validating Admission Policy Binding, true: create Validating Admission Policy Binding.")
+	DefaultGenerateVAPB = flag.Bool("default-create-vap-binding-for-constraints", true, "Create VAPBinding resource for constraint of the template containing VAP-style CEL source. Allowed values are false: do not create Validating Admission Policy Binding, true: create Validating Admission Policy Binding. Defaults to true.")
```

Comment:
We should not default alpha features to `true`

Reply:
After talking with Rita, the idea is that generation of VAP bindings will be governed by the template only (either via default value or explicitly setting intent).

This explains #3479. B/c this requires opt-in for the feature to activate, this SGTM